playbooks/roles/weave-network/files/weave-daemonset-k8s-v1.9.5.yaml

```diff
@@ -0,0 +1,118 @@
+kind: ClusterRole
```

Comment:
Looking around again - I found versions of the daemonset yaml on github release page:-)